src/app.ts

```diff
@@ -78,7 +75,13 @@ async function main() {
 }
 
 main().catch((err) => {
-  console.error(err);
+  const errorLog = {
+    '@level': 'error',
+    '@message': 'failed to start grafana-image-renderer',
+    'error': err.message,
+    'trace': err.stack,
+  }
+  console.error(JSON.stringify(errorLog));
```

Comment:
This error was actually logged as a "debug" log in plugin mode so that's why it was usually not displayed. This is fixing that and it should help us getting more info from user issues.